### PR TITLE
Add polkadot xcm pallet to khala runtime

### PR DESCRIPTION
**Pull request by tolak**
Stuff to support open HRMP channel with other parachains in Kusama network

### CODE REVIEW

File: runtime/khala/src/lib.rs [hunk 1]
```diff
@@ -252,6 +270,28 @@ construct_runtime! {
...
+                }
+                pallet_xcm::Call::force_xcm_version { .. }
+                | pallet_xcm::Call::force_default_xcm_version { .. }
+                | pallet_xcm::Call::force_subscribe_version_notify { .. }
+                | pallet_xcm::Call::force_unsubscribe_version_notify { .. }
+                | pallet_xcm::Call::send { .. } => {
+                    return true;
+                }
+                pallet_xcm::Call::__Ignore { .. } => {
+                    unimplemented!()
```
Comment (jasl) [inline runtime/khala/src/lib.rs:290]: if `unimplemented!()` triggered, the runtime will panic?
why not just `false`?

File: runtime/khala/src/lib.rs [hunk 2]
```diff
@@ -252,6 +270,28 @@ construct_runtime! {
...
 impl Contains<Call> for BaseCallFilter {
     fn contains(call: &Call) -> bool {
+        if let Call::PolkadotXcm(xcm_method) = call {
+            match xcm_method {
+                pallet_xcm::Call::execute { .. }
+                | pallet_xcm::Call::teleport_assets { .. }
+                | pallet_xcm::Call::reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_reserve_transfer_assets { .. }
+                | pallet_xcm::Call::limited_teleport_assets { .. } => {
+                    return false;
```
Comment (jasl) [inline runtime/khala/src/lib.rs:280]: looks no need `;`?

File: runtime/khala/src/lib.rs [hunk 3]
```diff
@@ -252,6 +270,28 @@ construct_runtime! {
...
+                }
+                pallet_xcm::Call::force_xcm_version { .. }
+                | pallet_xcm::Call::force_default_xcm_version { .. }
+                | pallet_xcm::Call::force_subscribe_version_notify { .. }
+                | pallet_xcm::Call::force_unsubscribe_version_notify { .. }
+                | pallet_xcm::Call::send { .. } => {
+                    return true;
```
Comment (jasl) [inline runtime/khala/src/lib.rs:287]: `;` is no need?

File: runtime/khala/src/lib.rs (hunk 1 above)
Comment (tolak) [reply runtime/khala/src/lib.rs:290]: Exactly, return false sounds better. Updated.

File: runtime/khala/src/lib.rs (hunk 2 above)
Comment (tolak) [reply runtime/khala/src/lib.rs:280]: We have `return` here, i think it's better to keep `;` at the end, even through compiler has not thrown error when i removed it.

File: runtime/khala/src/lib.rs (hunk 3 above)
Comment (tolak) [reply runtime/khala/src/lib.rs:287]: Ditto

File: runtime/khala/src/lib.rs (hunk 2 above)
Comment (h4x3rotab) [reply runtime/khala/src/lib.rs:280]: Better write like:

return match ... {
   a | b => true,
   c => false
}

Comment (tolak) [reply runtime/khala/src/lib.rs]: Done

**Review by h4x3rotab — APPROVED** [review]

**Pull request merged**